front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,168 @@
+  const refHulu = useRef();
+  const refApple = useRef();
+
+
+  const savePlatform = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div>
+      <p>Select the platform: </p>
+      <form onSubmit={(e) => savePlatform()}>
+
+        <input type="checkbox" id="netflix" value="Netflix" ref={refNetflix} />
+        <label for="netflix">Netflix  </label>
+        <input type="checkbox" id="prime" value="Prime" ref={refPrime} />
+        <label for="prime">Prime  </label>
+        <input type="checkbox" id="hulu" value="Hulu" ref={refHulu} />
+        <label for="hulu">Hulu  </label>
+        <input type="checkbox" id="apple" value="Apple" ref={refApple} />
+        <label for="apple">Apple TV+  </label><br /><br />
+        <input type="submit" value="Save Platform" />
+      </form>
+    </div>
+  )
+}
+
+/*the component displays and stores general info about the show*/
+
+const Description = ({ genre, description, total_episodes, is_movie }) => {
+  const refDescription = useRef();
+  const refIsMovie = useRef();
+  const refGenre = useRef();
+  const refTotalEpisodes = useRef();
+
+  const saveMovieData = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div class="description">
+      <br /><br />
+      <label for="genre">Genre:  </label>
+      <input type="text" id="genre" value={genre} ref={refGenre} readonly /><br /><br />
+      <label for="description">Description: </label>
+      <textarea id="description" value={description} ref={refDescription} readonly /><br /><br />
+      <label for="total_episodes">Total Episodes: </label>
+      <input type="text" id="total_episodes" value={total_episodes} ref={refTotalEpisodes} readonly /><br /><br />
+      <label for="isMovie">Is it a movie? </label>
+      <input type="text" id="isMovie" value={is_movie} ref={refIsMovie} readonly /><br /><br />
+    </div>
+  )
+}
+
+/*the component calls other components and displays comprehensive info about the show; 
+values passed as parameters to the main components are generated by the mock data API */
+
+const IndividualShow = (props) => {
+
+  const refTitle = useRef();
+  const refCover = useRef();
+  const returnToShows = (() => {
+    window.location.href = '/my-shows/'
+  });
+  const addToWatched = (() => {
+
+    window.location.href = '/my-shows/'
+  });
+
+
+  let [show, setShow] = useState([]);
+
+  useEffect(() => {
+    //temporary variable to be replaced
+    let showInfo = [];
+    let show_id = "54";
+    //axios.get(`https://my.api.mockaroo.com/shows/${show.id}.json?key=3010e030`)
+    axios.get(`https://my.api.mockaroo.com/shows/${show_id}.json?key=3010e030`)
+      .then((response) => {
+        showInfo.push(response.data);
+        console.log(showInfo);
+        console.log(show);
+        setShow(showInfo);
+      })
+  }, [])
+
+
+  return (
+    <>
+      <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
```

Comment:
This should be the `Header` component

Reply:
yup, I just noticed the new changes and made a pull request